spec/functional/resource/cron_spec.rb

```diff
@@ -80,6 +80,16 @@ def cron_should_not_exists(cron_name)
       5.times { new_resource.run_action(:create) }
       cron_should_exists(new_resource.name, new_resource.command)
     end
+    
+    # Test cron for day of week
+    weekdays = { Mon: 1, tuesday: 2, '3': 3, 'thursday': 4, 'Fri': 5, 6 => 6 }
+    weekdays.each do|key, value|
+      it "should create crontab entry and set #{value} for mon as weekday" do
```

Comment:
for `mon`? left over? You're using more than mondays...

Reply:
@jaymzh yeah I missed to update it. I have updated.